stackanetes-services/searchlight/searchlight-init.yml

```diff
@@ -0,0 +1,17 @@
+type: generic/job
+name: searchlight-init
+command: bash /tmp/searchlight-init.sh
+label: openstack-control-plane
+image: centos-binary-kolla-toolbox
```

Comment:
please remove that redundant prefix "centos-binary" from image. Prefix is configurable. If u use kolla-toolbox you don't even need to specify image name it's default value for jobs.